cfme/cloud/flavor.py

```diff
@@ -108,11 +143,66 @@ class Flavor(BaseEntity, Taggable):
     name = attr.ib()
     provider = attr.ib()
 
+    def delete(self, wait=True):
```

Comment:
I suggest passing ```cancel=False``` as a param here, which will give the capability of how the alert popup is handled.